src/gui/raster/qgsrasterlayerproperties.cpp

```diff
@@ -1609,83 +1600,17 @@ void QgsRasterLayerProperties::removeSelectedMetadataUrl()
 // Next four methods for saving and restoring qml style state
 //
 //
-void QgsRasterLayerProperties::loadDefaultStyle()
-{
-  bool defaultLoadedFlag = false;
-  QString myMessage = mRasterLayer->loadDefaultStyle( defaultLoadedFlag );
-  //reset if the default style was loaded OK only
-  if ( defaultLoadedFlag )
-  {
-    syncToLayer();
-  }
-  else
-  {
-    //otherwise let the user know what went wrong
-    QMessageBox::information( this,
-                              tr( "Default Style" ),
-                              myMessage
-                            );
-  }
-}
 
 void QgsRasterLayerProperties::saveDefaultStyle()
```

Comment:
Shall we not mark the old methods as deprecated ?

Reply:
Done!